src/components/canvas-tools/geometry-tool.tsx

```diff
@@ -83,8 +85,12 @@ export default class GeometryToolComponent extends React.Component<IProps, IStat
 
   private pointerDownHandler = (evt: any) => {
     const { board } = this.state;
+    const { model } = this.props;
+    const { ui } = this.stores;
     if (!board) { return; }
 
+    ui.setSelectedTile(model);
```

Comment:
The first click doesn't seem to take selection from the text component

Reply:
There are several issues with clicking on a geometry tile that I decided not to spend time on. The fact that clicking creates points is a temporary solution anyway.